### PR TITLE
Fix instruction when app is missing enableFlutterDriverExtension

### LINKED ISSUE #3171: Incorrect function name in output from flutter driver
Currently printing " To enable the driver extension call registerFlutterDriverExtension first"

Expected to print " To enable the driver extension call enableFlutterDriverExtension() first, and import import 'package:flutter_driver/driver_extension.dart';" and also tell me what library to import.

Or, just point the user to a URL that has all the good stuff.

### PR DESCRIPTION
**Pull request by sethladd**
This is a stop-gap fix. We plan for a more robust fix later.

In the meantime, closes https://github.com/flutter/flutter/issues/3171

### CODE REVIEW

**Comment by sethladd** [comment]
cc @yjbanov

File: packages/flutter_driver/lib/src/driver.dart [hunk 1]
```diff
@@ -118,8 +118,9 @@ class FlutterDriver {
       if (signal == 'timeout') {
         throw new DriverError(
           'Timed out waiting for Flutter Driver extension to become available. '
-          'To enable the driver extension call registerFlutterDriverExtension '
-          'first thing in the main method of your application.'
+          'Ensure your application (lib/main.dart) imports '
```
Comment (yjbanov) [inline packages/flutter_driver/lib/src/driver.dart]: More precisely: `'Ensure your test app imports '`

A test app is not necessarily `lib/main.dart`

**Comment by yjbanov** [comment]
LGTM

File: packages/flutter_driver/lib/src/driver.dart (hunk 1 above)
Comment (sethladd) [inline packages/flutter_driver/lib/src/driver.dart]: Thanks. Done.

**Comment by sethladd** [comment]
whoa. too many commits. closing.

**Pull request closed without merging**